File: Source Code/Geo-Tagging/Geo-tagging/Tag+CoreDataProperties.m
//
//  Tag+CoreDataProperties.m
//  
//
//  Created by Ankit on 24/02/19.
//
//

#import "Tag+CoreDataProperties.h"

@implementation Tag (CoreDataProperties)

+ (NSFetchRequest<Tag *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Tag"];
}

@dynamic strImageName;
@dynamic index;
@dynamic latitude;
@dynamic longitude;

@end

File: Source Code/Geo-Tagging/Geo-tagging/Geo-tagging/Core Data Model/Tag+CoreDataProperties.swift
//
//  Tag+CoreDataProperties.swift
//  
//
//  Created by Ankit on 24/02/19.
//
//

import Foundation
import CoreData


extension Tag {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Tag> {
        return NSFetchRequest<Tag>(entityName: "Tag")
    }

    @NSManaged public var title: String?
    @NSManaged public var strImagePath: String?

}
